doc/04-Getting-started.md

```diff
@@ -28,6 +28,67 @@ this is an advanced task you should not tackle at the early beginning.
 
 Take some time to really understand how to work with Icinga Director first.
 
+First steps on the Frontend
+====================================
+Create a host
+------------------
+
+First we create a `host template` which we will name `director-generic-host`. The default check we will use for a Host is `hostalive`.
+
+Click on the interface on Icinga Director > Host Objects > Add Host and `add a template`. For details have a look at the screenshots below:
+
+![add host section](screenshot/director/04-getting-started/01-add_a_host.png)
+
+![add host template](screenshot/director/04-getting-started/02-add_host_template.png)
+
+Now we can add our first host. Go to Icinga Director > Host Objects > Add Host and choose `Object`:
+
+![add host object](screenshot/director/04-getting-started/03-add_a_host_object.png)
+
+Deploy your Config
+------------------
+
+It is time to deploy our first configuration. Click on 
+Deployments/ History > Render config > Deploy to master
+
+![deploy config](screenshot/director/04-getting-started/04-render_and_deploy_a_config.png)
+
+It can be that you get an error while deploying your config. This is highlighted by the red X. Read the `Startup log` carefully and if it is a duplicate declaration, comment out the mentioned lines in your already existing config. Do not restart or reload the master while doing that. This would result in a breakdown of the API and won't make it possible to redeploy again. After canging the config, try redeploying again. 
+
+![deployment details](screenshot/director/04-getting-started/05-deployment_details.png)
+
+Go on
+-----
+
```

Comment:
That's too vague. We need at least examples for commands and services. And this is not a migration documentation, mentioning existing objects is therefore confusing at this point.